src/components/molecules/ImageInput/ImageInput.tsx

```diff
@@ -93,12 +107,19 @@ export const ImageInput: React.FC<ImageInputProps> = ({
           accept={ACCEPTED_IMAGE_TYPES}
           onChange={handleFileInputChangeWrapper}
           className="default-input"
+          ref={inputFileRef}
         />
+
         {loading && <ImageOverlay>processing...</ImageOverlay>}
         {remoteUrlInputName && (
           <input type="hidden" ref={register} name={remoteUrlInputName} />
         )}
       </div>
+      {isInputHidden && (
+        <Button onClick={onButtonClick} variant="primary">
+          Upload banner
+        </Button>
+      )}
       {errorMessage && <span className="input-error">{errorMessage}</span>}
```

Comment:
Good place to have something like https://github.com/sparkletown/sparkle/tree/fix/create-venue/src/components/atoms/AutoSpan at hand